bw_add_sshkeys.py

```diff
@@ -202,17 +217,21 @@ def ssh_add(session: str, item_id: str, key_id: str) -> None:
     )
     ssh_key = proc_attachment.stdout
 
+    if key_pw:
+        envdict = dict(os.environ, SSH_ASKPASS=os.path.realpath(__file__), SSH_KEY_PASSPHRASE=key_pw)
```

Comment:
what about doing an "echo $SSH_KEY_PASSPHRASE" instead of calling this script again?

Reply:
This wont work, as `ssh-add` expect an executable and it wont run `SSH_ASKPASS` through shell to parse arguments: 
```
$ cat ~/.ssh/test_rsa | SSH_ASKPASS="echo 123" ssh-add -                                                                                                                                     
ssh_askpass: exec(echo 123): No such file or directory
```